app/assets/stylesheets/hosts.scss

```diff
@@ -51,6 +51,17 @@
       border-bottom-right-radius: 4px;
     }
   }
+
+  .override-param {
+    text-decoration: line-through;
+    a, i {
+      text-decoration: none;
+    }
+    .btn[data-tag]{
+      display: none;
+    }
+  }
+
```

Comment:
Simplified the code and scoped to the `#params` tab. Also hide the button by css when needed.